developer-docs/upgrading_kubernetes.md

```diff
@@ -126,42 +126,66 @@ Be sure to review the rest of the sections as some of them may become irrelevant
 
 This step is specific to Rancher and serves to update Rancher's [Kontainer Driver Metadata](https://github.com/rancher/kontainer-driver-metadata/).
 
-* Create a PR in the latest [KDM](https://github.com/rancher/kontainer-driver-metadata/) dev branch to update the kubernetes versions in channels.yaml. The PR should consist of two commits. The first being the change made to channels.yaml to update the kubernetes versions. The second being go generate. To do this, run `go generate` and commit the changes this caused to data/data.json. Title this second commit "go generate".
-    * Please note if this is a new minor release of kubernetes, then a new entry will need to be created in `channels-rke2.yaml`. Ensure to set the min/max versions accordingly. If you are not certain what they should be, reach out to the team for input on this as it will depend on what Rancher will be supporting.
-    * As of v1.21.4 and above, every new release minor or patch requires a new entry be created in `channels-rke2.yaml`. It is possible to build off the server, agent, and chart arguments defined in other entries. For example, v1.21.4 has server args defined as follows:
-        ```
-        - version: v1.21.4+rke2r2
-        minChannelServerVersion: v2.6.0-alpha1
-        maxChannelServerVersion: v2.6.99
-        ...
-        charts: &charts-v1
-        rke2-cilium:
-            repo: rancher-rke2-charts
-            version: 1.9.808
-        ```
-        A later version can point to those arguments with no change:
-        ```
-        - version: v1.21.5+rke2r1
-        minChannelServerVersion: v2.6.0-alpha1
-        maxChannelServerVersion: v2.6.99
-        ...
-        charts: *charts-v1
-        ```
-        Or a later version can point to those arguments and create a new value with modification:
-        ```
-        - version: v1.21.5+rke2r1
-        minChannelServerVersion: v2.6.0-alpha1
-        maxChannelServerVersion: v2.6.99
-        ...
-        charts: &charts-v2
-        <<: *charts-v1
-        harvester-cloud-provider:
-            repo: rancher-rke2-charts
-            version: 0.1.200
-        ```
-
-* Create a backport PR in any additional dev branches as necessary.
-* The PRs should be merged in a timely manner, within about a day; however, they do not need to be merged before RC releases and they typically do not need to block the final release.
+Create a PR in the latest [KDM](https://github.com/rancher/kontainer-driver-metadata/) dev branch to update the kubernetes versions in channels.yaml. 
```

Comment:
Should we call out dev-v2.5 and dev-v2.6 specifically, since we need to do both for most release cycles?

Reply:
Sure we can always change the values when 2.7 eventually comes out.